src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -139,13 +153,22 @@ public function registerContainerConfiguration(LoaderInterface $loader)
     public function loadRoutes(LoaderInterface $loader)
     {
         $file = (new \ReflectionObject($this))->getFileName();
+        /* @var RoutingPhpFileLoader $kernelLoader */
```

Comment:
`/**`

Reply:
the codebase uses both styles, that's doesn't make any diff :)